source/client/ShowTable.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import textwrap
+import client.UIConstants as UIC
+from common.Card import Card
+
+
+class ShowTable:
+    """This class displays the public information -- cards melded, etc,...    """
+
```

Comment:
So - we can merge this with TableView when we have a chance to work together. to let it get notifications and such - but this is fine for now since it works.